plenum/server/consensus/ordering_service.py

```diff
@@ -1541,6 +1541,8 @@ def _order_3pc_key(self, key):
     def _add_to_ordered(self, view_no: int, pp_seq_no: int):
         self.ordered.add(view_no, pp_seq_no)
         self.last_ordered_3pc = (view_no, pp_seq_no)
+        if self._data.prev_view_prepare_cert == pp_seq_no:
```

Comment:
We don't call this method if pp_seq_no is already ordered. So, we will never call `MasterReorderedAfterVC` and start ordering on backups if all batches from NEW_VIEW are already ordered.